Makefile

```diff
@@ -163,6 +163,11 @@ check-docs: $(EMBEDMD) $(LICHE) build
 	@$(LICHE) --recursive docs --exclude "(cloud.tencent.com|alibabacloud.com)" --document-root .
 	@$(LICHE) --exclude "(cloud.tencent.com|goreportcard.com|alibabacloud.com)" --document-root . *.md
 
+# check-markdown-links checks if markdown files have broken links.
+.PHONY: check-markdown-links
+check-markdown-links:
+	./scripts/marker
```

Comment:
I'm not a fan of checking in binaries like that. I think it's fair to ask people to install it themselves.

Reply:
I understand the desire to make this easy to use, but unfortunately it’s not portable. What if someone isn’t running Linux on AMD64?